Documentation/kubernetes-upgrade.md

```diff
@@ -65,9 +65,9 @@ The kube-proxy is run as a "static pod". To upgrade the pod definition, simply m
 ## Example Upgrade Process:
 
 1. Prepare new pod manifests for master nodes
-1. Prepare new pod manifests for worker nodes
-1. For each master node:
+2. Prepare new pod manifests for worker nodes
+3. For each master node:
     1. Back up existing manifests
-    1. Update manifests
-1. Repeat same steps above for each worker node
+    2. Update manifests
+4. Repeat item 3 for each worker node
```

Comment:
nit: This makes it more clear when looking at the raw markdown, but the rendered output won't necessarily follow this numbering (why I tend to use "1." for every list item).